### ISSUE TITLE
Launching Cmder.exe

### ISSUE THREAD

**Issue by neil-s** [wbond/sublime_terminal#99]
I spent some time figuring out how to launch Cmder on Windows correctly (in the right directory). For anyone else looking for this, the key is to add the following user settings for this plugin:

```
{
        // Of course, replace the path with your own path to cmder.exe
	"terminal": "C:\\Program Files\\cmder_mini\\cmder.exe",

        // %CWD% gets replaced by the path the command prompt is meant to be opening in (current folder or not)
	"parameters": ["/START", "%CWD%"]
}
```

That's it. No question or bug report, just didn't know how else to contribute this tip.

**Comment by twolfson** [comment]
Woot, thanks for the tip. Variations on this seem to be a recurring issue. Maybe we should add common examples to our README? /cc @wbond

**Comment by wbond** [comment]
I'm not opposed to adding readme's on this.

As I have mentioned in other issues, the way forward for Terminal is to rewrite it based on newterm (which already automatically handles cmder - https://github.com/codexns/newterm/blob/master/all/newterm/_win.py#L143-L180).

Unfortunately I need to pay the bills, so the rewrite is back burner for me right now.

If someone else is interested in taking a stab at starting the process of utilizing newterm, I think the most complicated part will be helping users transition from old settings to whatever new settings will look like so they can work with newterm.

**Comment by twolfson** [comment]
With respect to the README, I will submit a PR by the end of next weekend (unless someone beats me to it).

With respect to newterm, I am going to ask for more info on the main thread (#94).

**Comment by neil-s** [comment]
I would have added it to the README myself, but I can't find a repository for your website, where the rest of the documentation is. If it's open and you point me to it, I'd be happy to have a stab at writing the content and getting your editorial feedback. I've never contributed to open source before, so writing some mini-docs might be perfect to get me acquainted with writing PRs and so forth. Although if the newterm transition is going to make this configuration redundant soon, let me know if we want to bother in the mean time?

In the longer term, I'd be keen to try and help with the newterm transition too.

Longer term, is there also any plan to support a terminal embedded into the Sublime interface, similar to the Atom package (https://atom.io/packages/terminal-plus)?

**Comment by wbond** [comment]
Oh yeah, my website. I should just redirect the terminal page to GitHub for now. Adding it to the readme in this repo is where it should go.

In terms of embedding the terminal into Sublime Text, I don't think that will ever happen. Mostly because it is fraught with peril. If you look at terminal programs, there are so many things they need to handle to act like a proper terminal. Like key codes (ctrl-c, etc), terminal colors, the bell, hooking up to a tty (OS X/Linux) and a console (Windows) for non-printed password input, etc.

There are, however, many packages that allow you to run basic commands and print the output in a view. This is probably about as far as a terminal inside of Sublime Text should really go, IMO.

**Comment by twolfson** [comment]
@wbond You could also forward to packagecontrol.io which renders the README content as well.

@neil-s The README is definitely a good place to start. Feel free to rewrite it and submit a PR =) We have been wanting to move to Github Flavored Markdown but for me, it got deprioritized in my personal queue.

Fwiw, there was a PR up for it but the author became unresponsive: https://github.com/wbond/sublime_terminal/pull/47

I left some notes on it about the style of the markdown itself (e.g. using `#` for consistency in wbond's repos).

**Comment by twolfson** [comment]
It's been 9 days since the last comments. I'm going to write up a PR to replace the README.

**Comment by twolfson** [comment]
#101 is now open which documents this use case =)

**Comment by neil-s** [comment]
Whoops, sorry, got distracted because I suddenly had to move countries (silly visa rules). Thanks for stepping in and picking it up!

**Comment by twolfson** [comment]
We have released #101 in `1.10.1` =)
